core/tests/protractor_utils/waitFor.js

```diff
@@ -18,6 +18,9 @@
  */
 
 var until = protractor.ExpectedConditions;
+var fs = require('fs');
+var path = require('path');
+var DPATH = path.resolve(__dirname, '../downloads');
```

Comment:
Since we use this in two files, let's move this to the protractor constants file. Also, could you make the name a bit more verbose, e.g. `DOWNLOAD_PATH`? It's a little hard to understand now

Reply:
I've renamed it and moved it into `ProtractorConstants.js`